group-coordinator/src/main/java/org/apache/kafka/coordinator/group/classic/ClassicGroup.java

```diff
@@ -857,6 +857,7 @@ public void validateOffsetCommit(
             throw Errors.UNKNOWN_MEMBER_ID.exception();
         }
 
+        // TODO: A temp marker. Will remove it when the pr is open.
         if (!isTransactional && isInState(COMPLETING_REBALANCE)) {
```

Comment:
Not sure why we only check `COMPLETING_REBALANCE` but not `PREPARING_REBALANCE`?